bin/soaringspot.js

```diff
@@ -211,7 +211,7 @@ async function update_pilots(class_url,classid,classname,keys) {
     });
 
     // remove any old pilots as they aren't needed, they may not go immediately but it will be soon enough
-    t.query( escape`DELETE FROM PILOTS WHERE class=${classid} AND registereddt < DATE_SUB(NOW(), INTERVAL 15 MINUTE)`)
+    t.query( escape`DELETE FROM pilots WHERE class=${classid} AND registereddt < DATE_SUB(NOW(), INTERVAL 15 MINUTE)`)
```

Comment:
interesting this is different, I must have some backwards compatibility settings on my mysql server!